src/validation/rules/DefaultValuesOfCorrectType.js

```diff
@@ -27,10 +27,12 @@ export function defaultForNonNullArgMessage(
 export function badValueForDefaultArgMessage(
   varName: any,
   type: any,
-  value: any
+  value: any,
+  verboseErrors?: [any]
 ): string {
+  var message = verboseErrors ? [ '', ...verboseErrors ].join(' ') : '';
```

Comment:
rather than duplicating the verboseErrors array here, you could replace this with:

```js
var message = verboseErrors ? ' ' + verboseErrors.join(' ') : '';
```